### PR TITLE
Prism landing page

### PR DESCRIPTION
**Pull request by XVincentX**
SO-242

### CODE REVIEW

File: netlify/landings/prism.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+path: /prism
+title: Open Source Mock Server
+color: purple-darker
+relatedTags:
+  - blog-mocking
+hero:
+  subtitle: Build APIs and accelerate API development with instant mock servers
+  title: Write the spec. Prism will do the rest.
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]:
```suggestion
  title: Free, Realistic, HTTP Mocking for API Descriptions
```

Spectral website is talking about API Description Documents in line with a bunch of API peoples decision to start talking about API Description Documents and API Description Formats instead of API Specifications.

File: netlify/landings/prism.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+featureSection:
+  actionBar:
+    buttons:
+      - href: 'https://github.com/stoplightio/prism'
+        title: Try Prism now
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]:
```suggestion
        title: Get Started
```
"Try" makes it sound like a trial.

File: netlify/landings/prism.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    enabled: true
+    text: 'Want to mock your API?'
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]:
```suggestion
    text: Want to create a mock API?
```

Mock your API makes it sound like creating a fake impl of an existing API, which is a valid use case but not the primary one, and not the first thing that people think of when they think about mocking: parallelizing development efforts by getting feedback before they waste time writing code.

File: netlify/landings/prism.yaml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  features:
+    - description: >-
+        The waterfall model does not work for APIs. Learning at the very end what your
+        potential consumers think about your software can be source of huge problems.
+        With Prism you can move the feedback stage at the design phase of your product by
+        being able to provide a real server to use and play with.
+      image: /images/mocking_instant.png
+      shortName: Quick iterations
+      title: Quickly iterate on your API
+      titleURL: /mocking/server
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]: What is `titleURL` all about? 

Is this an existing website, or are we creating new URLs?

File: netlify/landings/prism.yaml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    - description: >-
+        Prism validates the data you feed it with and all the data it returns. If you have a
+        wrong example in the specification, Prism will let you know; if you request is not compliant with the rules
+        of your API specification, Prism will return an error message. It will never yell to you, we promise.
+      image: /images/mocking_request_validation.png
+      shortName: Validation
+      title: Input and Output validation
+
+    - description: >-
+        In case you're wondering why Prism has responded a request in such way, you can see
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]:
```suggestion
        In case you're wondering why Prism has responded to a request in such way, you can see
```

File: netlify/landings/prism.yaml (hunk 4 above)
Comment (XVincentX) [reply netlify/landings/prism.yaml]: It's an existing URL, I've removed it.

File: netlify/landings/prism.yaml [hunk 6]
```diff
@@ -0,0 +1,78 @@
+path: /prism
+title: Open Source Mock Server
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]:
```suggestion
title: Prism - The Free Open Source HTTP Mock Server
```

File: netlify/landings/prism.yaml [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+color: purple-darker
+relatedTags:
+  - blog-mocking
+hero:
+  subtitle: Build APIs and accelerate API development with instant mock servers
```
Comment (philsturgeon) [inline netlify/landings/prism.yaml]:
```suggestion
  subtitle: Accelerate API development with realistic mock servers powered by API descriptions
```

**Comment by philsturgeon** [comment]
@XVincentX any luck with images? where are you looking and for what sort of thing?

**Comment by lottamus** [comment]
If you want the page to live at `https://stoplight.io/prism` you'll need to remove this redirect: https://github.com/stoplightio/stoplight.io/blob/develop/public/_redirects#L12

**Comment by lottamus** [comment]
> If you want the page to live at https://stoplight.io/prism you'll need to remove this redirect: https://github.com/stoplightio/stoplight.io/blob/develop/public/_redirects#L12

I went ahead and removed it for you on this PR.

Not sure if you know but you can see the preview here: https://deploy-preview-930--stoplightio.netlify.com/prism/

File: netlify/landings/prism.yaml [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+      shortName: Quick iterations
+      title: Quickly iterate on your API
+
+    - description: >-
+        Mocking the returned data is important — but static examples can bias the way you write
+        the code and test the API. Prism can generate dynamic examples based on the provided schema,
+        so that you get valid, but still different data at every call. Figure out the edge cases
+        while designing the API.
+      image: /images/mocking_virtualization.png
+      shortName: Intelligent example generation
```
Comment (tbarn) [inline netlify/landings/prism.yaml]: "Static and Dynamic Examples" makes a lot more sense to me here. It's closer to the lingo people know around mocking.

File: package.json [hunk 9]
```diff
@@ -54,7 +54,7 @@
     "markdown-it-anchor": "5.0.x",
     "markdown-it-toc-done-right": "2.0.x",
     "markdown-it-video": "0.6.x",
-    "node-sass": "4.10.x",
+    "node-sass": "4.12.x",
```
Comment (XVincentX) [inline package.json]: In case you're wondering this is required so that works on Node 12

**Comment by tbarn** [comment]
Just a heads up, I didn't get to finish going through all the descriptions today. There's more edits to be made, but I'll work on it a little bit through out the week. 
 
@adamd still feel free to review what is here though.

File: package.json (hunk 9 above)
Comment (lottamus) [inline package.json]: I believe this is breaking server side rendering. I'll open a jira ticket to investigate upgrading the this repo's deps.

Can you revert and set the node engines to ">=8.0.0 <11.0.0"?

Comment (lottamus) [reply package.json]: I went ahead and took care of this for you already

**Comment by tbarn** [comment]
@XVincentX  Can we get the latest changes pulled into this?

**Comment by tbarn** [comment]
I'm going to add it to the Resources dropdown with a [server icon](https://fontawesome.com/icons/server?style=solid)

File: netlify/landings/prism.yaml [hunk 10]
```diff
@@ -0,0 +1,74 @@
+path: /prism
+title: Prism - Open Source HTTP Mock Server | Stoplight
+color: purple-darker
+relatedTags:
+  - blog-mocking
+hero:
+  subtitle: Accelerate API development with realistic mock servers powered by API descriptions
+  title: Prism, an Open Source HTTP Mock Server for API Descriptions
```
Comment (tbarn) [inline netlify/landings/prism.yaml]: What do you think about making the title? 

`Prism, an Open Source HTTP Mock Server`

I think the subtitle does a good job of expanding and the title feels.... long?

**Comment by tbarn** [comment]
Looking really good! The last thing I need to help with is images. I got some ideas. Similar to what I did for Spectral.

File: netlify/landings/prism.yaml (hunk 10 above)
Comment (philsturgeon) [reply netlify/landings/prism.yaml]: Sounds good!

**Comment by philsturgeon** [comment]
Validation could be prettified JSON showing invalid payload, of a user sending a numeric string when it should be new an integer, and missing a required field, the usual sort of integration mistakes.

File: netlify/landings/prism.yaml [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+hero:
+  subtitle: Accelerate API development with realistic mock servers powered by API descriptions
+  title: Prism, an Open Source HTTP Mock Server
+  ctas:
+    - color: indigo
+      href: 'https://github.com/stoplightio/prism/'
+      name: View it on GitHub
+      icon: ['fab', 'github']
+    - color: indigo
+      href: 'https://www.youtube.com/watch?v=IJtFI9mkZ8Y'
```
Comment (lottamus) [inline netlify/landings/prism.yaml]: @tbarn can you update this with the correct youtube video?

If we don't have one, we can remove it

Comment (tbarn) [reply netlify/landings/prism.yaml]: Yeah, still waiting on getting the video back... might add it later in the day if we don't have it at 10am.

**Comment by XVincentX** [comment]
@philsturgeon I'm removing the link, we'll put it back once we have it from Ryan

**Comment by XVincentX** [comment]
I'm merging this into the other PR so we have one thing to take care of
